portal/server/controllers/dashboard/facilities/facilities-filters-query.js

```diff
@@ -16,17 +16,30 @@ const dashboardFacilitiesFiltersQuery = (
   user,
 ) => {
   const query = {};
+  let dashboardFilters = filters;
+
   // if user is admin, then deal.bank.id should not be set so cannot see all
   if (!isSuperUser(user)) {
     query.$and = [
       { 'deal.bank.id': user.bank.id },
     ];
   }
 
-  if (filters.length) {
+  const filtered = [];
+  // removes _crsf from dashboardFilters
+  /* eslint-disable no-unused-vars */
```

Comment:
Will need to review explicit disablement of linting rules.

Reply:
done